packages/ag-grid-addons/src/agGridStyles/useAgGridStyles.ts

```diff
@@ -0,0 +1,14 @@
+import { useEffect } from 'react';
+import { agStyles } from './output/styles';
+
+export const useAgGridStyles = () => {
+  useEffect(() => {
+    const style = document.createElement('style');
```

Comment:
give the element a name or id since multiple components might call for this, skip if already in document